### PR TITLE
Payment details

### PR DESCRIPTION
**Pull request by sadlerw**
* add showDetails and toggle to the detail view
* add the html to the case detail view
* add default payment info to the state
* add the sequence
* css for the paygov button
* flag icon stuff

screen shot 2018-11-13 at 2 58 04 pm

### CODE REVIEW

File: web-client/src/components/CaseDetail.jsx [hunk 1]
```diff
@@ -22,6 +30,70 @@ export default connect(
...
+                  </div>
+                  <p>
+                    <i>
+                      Note: it may take up to X days for your payment to appear
+                      online.
+                    </i>
+                  </p>
+                </div>
+                <div className="usa-width-one-half">
+                  <h4>Can&rsquo;t afford to pay the fee?</h4>
```
Comment (kkoskelin) [inline web-client/src/components/CaseDetail.jsx]: Please replace HTML entity with actual UTF-8 character whenever possible.

**Review by kkoskelin — APPROVED** [review]
Looks good. re: entity, let's fix it before W finds it.  ;)

File: web-client/src/components/CaseDetail.jsx (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/components/CaseDetail.jsx]: Agreed, this came up during our last PR to 18F.

**Review by ericsorenson — APPROVED** [review]
🤩

**Pull request merged**